test/e2e/space_autocompletion_test.go

```diff
@@ -125,6 +90,30 @@ func TestAutomaticClusterAssignment(t *testing.T) {
 		})
 	})
 
+	t.Run("set low capacity threshold and expect that space will have default tier, but won't have target cluster so it won't be provisioned", func(t *testing.T) {
+		// given
+		hostAwait.UpdateToolchainConfig(
+			testconfig.CapacityThresholds().ResourceCapacityThreshold(1),
```

Comment:
what about `-1` to be sure that conditions such as https://github.com/codeready-toolchain/host-operator/blob/master/pkg/capacity/manager.go#L20 and https://github.com/codeready-toolchain/host-operator/blob/master/pkg/capacity/manager.go#L45 return false, what ever the current number of spaces on the cluster when the test is executed?

Reply:
makes sense, if this makes i more clear, I can update the test.